src/components/aeIdentity/aeIdentity.scss

```diff
@@ -0,0 +1,139 @@
+@import "./../variables";
+
+.ae-identity {
+  top:auto;
+  bottom:10px;
+  box-sizing:border-box;
+  zoom: 1.1;
+  padding:15px 21px;
+  color:white;
+  z-index:5;
+  margin:0 auto;
+  text-align: left;
+  width: 335px;
+  box-shadow: 0 4px 8px 0 rgba(60, 60, 60, 0.1);
+  border-radius: 12px;
+  background-image: linear-gradient(122deg, #6f53a1, $exciting);
+
+  &.active{
+    background-image: linear-gradient(119deg, #f22f70, #42255c);
+  }
+
+  &.collapsed{
+  }
+
+  .footer {
+    margin-top: 18px;
+
+    .buttons {
+      position: relative;
+      width: 100%;
+      display: flex;
+      flex-wrap: wrap;
+      align-items: right;
+      justify-content: flex-end;
+      margin-top: 12px;
+    }
+
+    .ae-button {
+      margin-left: 5px;
+    }
+  }
+
+
+  .avatar {
+    width: 23px;
+    height: 23px;
+
+    &.ae-identity-avatar {
+      margin-bottom: 0px;
+    }
+  }
+
+  .flex-row{
+    position:relative;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .chunked-address {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 10px;
+    margin-left: -10px;
+
+    .chunk {
+      display: inline-block;
+      margin: 5px 0 0 10px;
+      flex-grow: 1;
+      width: calc(100% * (1/3) - 10px - 1px);
+      justify-content: space-between;
+
+      font-family: 'Roboto Mono', monospace;
+      font-size: 17px;
+      font-weight: 500;
+      font-style: normal;
+      color: $plain;
+      line-height: 1.53;
+      letter-spacing: 3.1px;
+      font-stretch: normal;
+      text-align: left;
+    }
+  }
+}
+
+.ae-identity ,
+.ae-identity * {
+  transition:0.4s;
+}
+
+.ae-identity._active_no{
+  background-image: linear-gradient(to bottom, #aeaeae, #dcdcdc);
+}
+#app.home .ae-identity {
```

Comment:
I know this was introduced in a commit before, but please remove this. this is not the right location for this style. i think this actually belongs to a specific application.
component should be atomic and not assume existence of certain classes in parent components. component position should be controlled by the parent. its just not clear for a developer that uses a common name like `.home` why it changes the position of all ae-identity.